README.md

```diff
@@ -1,42 +1,35 @@
 ![Code for Baltimore](/docs/img/CfB.png)
 
-# ProjectTemplate
-The project template for Code for Baltimore github projects.  This project will help your project get started by giving you:
-* A standard issues template
-* A standard pull request
-* A standard docs folder with example documentation.
-* A big thumbs up :thumbsup:
-
 ## Documentation
-We've included a `docs` folder with a template [Tech Spec](/docs/Tech_Spec.md) and [Best Practices](/docs/Best_Practices.md) document, though using Github's Wiki capabilities is also a good idea. This will get you started with documenting your project.  Other documents and relevant information that has no other place can live in the `docs` folder.  Replace this paragraph with a brief breakdown of what you've included in your `docs` folder.
 
 ## Setup
 To run this locally the following software is required:
-*  [Python](https://www.python.org/)
-*  [Docker](https://docker.com) *optional*
-
-If on a Mac you can install Node and npm via Homebrew with `brew install python`
+*  [Python](https://www.python.org/) (version minimum 3.6)
+*  [Flask](https://palletsprojects.com/p/flask/)
+*  [Flask-Marshmallow](https://flask-marshmallow.readthedocs.io)
+*  [Flask-SQLAlchemy](https://flask-sqlalchemy.palletsprojects.com)
+*  [Docker](https://docker.com) (*for optional container image usage*)
```

Comment:
Maybe add a small bit about adding the SQL db

Reply:
The "test" database is not intended to be shipped. This is for dev testing only. Someone will need to figure out the proper seeding mechanism required once someone moves to an official database. Hopefully, no one is intending to run off a SQLITE3 on an actual production system due to concurrent writes causing DBLocks

Reply:
Fair enough, but it is a requirement to run the application at this time. I would just suggest adding something to explain that for now, but that's your call.